content/about/about.fr.md

```diff
@@ -0,0 +1,68 @@
+---
+title: à propos de
+description: "à propos de | Node.js"
+authors: XhmikosR, mikeal, fhemberger, Fishrock123, yous, tomgco, tniessen, SMcCandlish, saadq, Trott, Gornstats, piperchester, naoufal,  lpinca, j9t, bnoordhuis, harshadsabne, Chris911, benhalverson, AugustinMauroy
+category: à propos de
```

Comment:
`category` must be `about`

Reply:
Oh ok thanks

Reply:
I made the changes but the problem remains

Reply:
Can you check my PR for translation guidelines? I wasn’t expecting someone to contribute towards translations this soon 🙈

the title in this PR needs to be the same as the english one (so title and category fields should not be changed)

Reply:
oh yes, I just saw the pr oops my bad